docs/prometheus-integration.md

```diff
@@ -154,6 +154,10 @@ by flowPollInterval, a configuration parameter for the Agent.
 - **antrea_agent_denied_connection_count:** Number of denied connections
 detected by Flow Exporter deny connections tracking. This metric gets updated
 when a flow is rejected/dropped by network policy.
+- **antrea_agent_flow_exporter_to_flow_aggregator_count:** Number of Flow
+Exporters connected to Flow Aggregator. This metric gets updated when a
+connection is established or removed between Flow Exporter and Flow
+Aggregator.
```

Comment:
I am confused by this metric. I thought we would have something to count the number of reconnections to the Flow Aggregator, so something that would steadily increase over time.

Instead you seem to be counting the number of current active connections to the Flow Aggregator, which I imagine is either going to be 0 or 1? Isn't there a single Flow Exporter instance per Antrea Agent?

Reply:
I read Srikar's comment and things make more sense to me now. I agree with Srikar that some additional information may be helpful in the description.